src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -205,20 +205,13 @@ post_processing()
   if (base_type::post_processing())
     return true;
   
-  this->subiteration_counter=0;
   this->anatomical_sd.resize(anatomical_image_filenames.size());
 
   if (this->anatomical_image_filenames.size()!=sigma_m.size()){
```

Comment:
this test should be repeated in `set_up` but with `anatomical_prior_sptrs.size()`